src/core/annotation.js

```diff
@@ -1359,26 +1369,41 @@ class WidgetAnnotation extends Annotation {
     return [initialState.font, initialState.fontName, initialState.fontSize];
   }
 
-  _computeFontSize(font, fontName, fontSize, height) {
+  _computeFontSize(font, fontName, fontSize, height, lineCount) {
     if (fontSize === null || fontSize === 0) {
-      const em = font.charsToGlyphs("M")[0].width / 1000;
-      // According to https://en.wikipedia.org/wiki/Em_(typography)
-      // an average cap height should be 70% of 1em
-      const capHeight = 0.7 * em;
-      // 1.5 * capHeight * fontSize seems to be a good value for lineHeight
-      fontSize = Math.max(1, Math.floor(height / (1.5 * capHeight)));
+      if (lineCount === -1) {
+        fontSize = Math.round(0.8 * height * 10) / 10;
+      } else {
+        // Hard to guess how many lines there are
+        // let's say that fonsize is 10.
+        fontSize = 10;
+        let lineHeight = fontSize / 0.8;
+        let numberOfLines = Math.round(height / lineHeight);
```

Comment:
Shouldn't we use `Math.ceil` here? If there are 5.05 lines, this would round it down to 5, causing a part of the line to be cut off?

Reply:
In this case fontSize will be readjusted to for these 5 lines.